MiguelAValdesG/SpringDataJPA/src/test/java/com/bootcamp/springdatajpa/SpringDataJpaApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.bootcamp.springdatajpa;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class SpringDataJpaApplicationTests {
+
+  @Test
+  void contextLoads() {
```

Comment:
a test without an assert?

Reply:
This class was created automatically from Spring Initializr, I didn't see it, but for now I'll delete this method